editor/osm_editor.cpp

```diff
@@ -41,7 +41,7 @@ using std::move, std::make_shared, std::string;
 
 namespace
 {
-constexpr char const * kXmlRootNode = "mapsme";
+constexpr char const * kXmlRootNode = "omaps";
```

Comment:
I afraid that we can break existing user's edits file with this change.

Reply:
@vng how? The new file is bundled and will overwrite the old one. Only a few people who manually override it may see some errors. We can support their case also, of course, but it may be easier for them to edit the file.

Reply:
This is the user edits file, not config. See Editor::LoadEdits()

Reply:
If you want i can doing more tests on device, for example install older OM version and after install this version to check if we have some problems ?

Reply:
Here is edits.xml after adding dummy POI:
```
<?xml version="1.0"?>
<mapsme format_version="1">
  <mwm name="Belarus_Minsk Region" version="221119">
    <delete />
    <modify />
    <create>
      <node lat="54.0446163" lon="27.6597626" mwm_file_index="4293918720" timestamp="2022-12-09T18:58:28Z">
        <tag k="name:ru" v="xxx" />
        <tag k="amenity" v="bar" />
      </node>
    </create>
    <obsolete />
  </mwm>
</mapsme>
```

Reply:
Ok, I am not an expert in develpoment,
If you want i can remove this commit and add a commentary in editor.config, to just tell don't replace mapsme in tag

Reply:
It can be replaced, but the old one should also be supported for the Load operation. @vng thanks for pointing to that. Let's leave this change for a separate PR, it's more complex than I thought.